server/http/restapi/restapi.go

```diff
@@ -38,6 +38,7 @@ func Init(router *mux.Router, mm *pluginapi.Client, conf config.Service, proxy p
 	subrouter.HandleFunc(config.PathCall, a.handleCall).Methods("POST")
 	subrouter.HandleFunc(mmclient.PathSubscribe, a.handleSubscribe).Methods("POST")
 	subrouter.HandleFunc(mmclient.PathUnsubscribe, a.handleUnsubscribe).Methods("POST")
+	subrouter.HandleFunc(mmclient.PathRefreshBindings, a.handleRefreshBindings).Methods("POST")
```

Comment:
Can we simplify things for an app developer here by assuming that the bindings need to be reloaded every time an OAuth flow completes?

Reply:
Maybe the path here is to start with giving the developer the functionality first and move it to the framework once we understand the usage/needs better.

Reply:
How would the transition from the API to the framework work?